stubs/nova_resource.blade.php

```diff
@@ -18,16 +19,25 @@ class {{$model}}Resource extends Resource
 
     public static function label(): string
     {
-        return '{{\Illuminate\Support\Str::plural($model)}}';
+        return '{{Str::plural($model)}}';
     }
 
     public function fields(Request $request): array
     {
         return [
-            @foreach($fields as $fieldName => $fieldType)
+        @foreach($fields as $fieldName => $fieldOptions)
+            @php
+                $headline = Str::headline($fieldName);
+                $result = "{$fieldOptions['type']}::make('{$headline}')";
 
-            {{$fieldType}}::make('{{\Illuminate\Support\Str::of($fieldName)->ucfirst()->replace('_', ' ')}}')->sortable(),
-            @endforeach
+                echo ($fieldOptions['is_required'])
+                ? "\n\t\t\t{$result}
+                ->sortable()
+                ->required(),\n"
+                : "\n\t\t\t{$result}
+                ->sortable(),\n"
```

Comment:
could we use something like there?

```suggestion
                @if($fieldOptions['is_required'])
                ->required(),
                @endif
                ->sortable(),
```